drivers/ccpp/noahmpdrv.meta

```diff
@@ -96,6 +96,230 @@
   dimensions = ()
```

Comment:
As we discussed both files need to be pushed to NoahMP EMC repository too.

Reply:
> As we discussed both files need to be pushed to NoahMP EMC repository too.

Yes, they are now copied.